src/pokemon_storage_system.c

```diff
@@ -10210,3 +10247,14 @@ void UpdateSpeciesSpritePSS(struct BoxPokemon *boxMon)
     }
     sJustOpenedBag = FALSE;
 }
+
+u8 GetInPartyMenu(void)
```

Comment:
This function is unused

Reply:
I made it for determining if the pokemon was in the party or not from other files after the pokemon had been selected. Not sure if there was another way to determine this